lib/blueprinter/configuration.rb

```diff
@@ -1,3 +1,5 @@
+require 'json'
```

Comment:
This has to be removed. We purposefully did not `require 'json'` because we do not want to assume that people will be using the `json` lib. The `json` lib is very invasive, it will override basic ruby objects and add `to_json` methods to them.

It is up to users of blueprinter to require the json marshaller gems of their choosing. It's noted here: https://github.com/procore/blueprinter#installation

Reply:
You should just need to `require 'json'` in the spec_helper only for the specs.